app/volunteers/dashboard/events/page.tsx

```diff
@@ -64,8 +64,8 @@ export default function Dashboard() {
               <span>{event.date}</span>
             </CardDescription>
           </CardContent>
-          {eventStatus?.map((status) => (
-            <CardDescription>{status.status}</CardDescription>
+          {eventStatus?.map((status, index) => (
```

Comment:
If you follow the review against making this it's own array, you can use the `event_id` instead of `index` here, making it a better key, because then it would actually be unique to the events.

Only use `index` as key if your data does not mutate.[^1]

[^1]: https://stackoverflow.com/questions/59517962/react-using-index-as-key-for-items-in-the-list